### PR TITLE
Update to rails 6

### LINKED ISSUE #83: Update for Rails 6
firstdraft/appdev#161

### PR DESCRIPTION
**Pull request by jelaniwoods**
Resolve #83 

Update the template to configure Rails 6 apps in the decided way.

- adds flags
  ```
  --skip-action-mailbox
  --skip-action-text
  ```
- adds `config.hosts.clear` to `development.rb` [read more](https://www.fngtps.com/2019/rails6-blocked-host/)
- slightly update `READ.me`
- removes bootstrap links from `application.html.erb`
- uses `web_git` gem from [github](https://github.com/firstdraft/teachersmanual/blob/master/updating-old-projects-checklist.md#gems).

Clone, checkout jw-update-to-rails-6 branch, 
```bash
rails new rails-6-test-app --rc=<path to fd> -m <path to template.rb> 
```

The new app should be able to navigate to:
- `/` and see "Yay, you're on Rails
- `/git`
- `/admin` and login with `admin@example.com` and `password`

#### Note:
There are a couple temporary changes I've made so that review is as easy as possible.
In `template.rb`
```ruby
# ENV = :prod
ENV = :dev
```
This change is because we're in development and the changes aren't in `master` yet.

In `fd`
The removal of
```
--template=https://raw.githubusercontent.com/firstdraft/appdev_template/master/template.rb
```
For similar reasons. I will add them back after approval.

### CODE REVIEW

File: files/README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ## Standard Workflow
```
Comment (pmckernin) [inline files/README.md]: In the Standard Workflow, all of the items are numbered as 1.  Does this need to be descending order?

Comment (jelaniwoods) [reply files/README.md]: That's one way to write an ordered list in Markdown. It will be ascending but it already was previously so I left it that way.

**Comment by LoganDSPrice** [comment]
I apparently don't have access to the firstdraft/web_get repo so it fails to create a new app using the template...

**Comment by LoganDSPrice** [comment]
OK yeah I thought that was weird. Looks like theres a typo in `template.rb`, ln. 55

File: template.rb [hunk 2]
```diff
@@ -44,17 +44,15 @@ def render_file(filename)
...
   gem "awesome_print"
   gem "console_ip_whitelist", github: "firstdraft/console_ip_whitelist"
   gem "dotenv-rails"
   gem "grade_runner", github: "firstdraft/grade_runner"
   gem "pry-rails"
-  gem "sqlite3", "~> 1.3.6"
+  gem "sqlite3", "~> 1.4.1"
   gem "table_print"
-  gem "web_git"
+  gem "web_git", github: "firstdraft/web_get"
```
Comment (LoganDSPrice) [inline template.rb]: This is the aforementioned typo. Unless there's a sneaky repo named `web_get`

Comment (jelaniwoods) [reply template.rb]: Nice catch! I made this change last and didn't re-create a project to check.

**Comment by jelaniwoods** [comment]
@LoganDSPrice I've made the suggested changes. Could you take another quick look?

**Comment by LoganDSPrice** [comment]
I got the following error:
```
No such file or directory @ rb_sysopen - /Users/logandsprice/code/appdev/appdev_template/rails-6-test-app/app/assets/javascripts/application.js
```

There is, however, `app/javascript/packs/application.js`. Is it possible that despite skipping the webpack install, it's still placing the file where webpacker would expect it?

**Comment by jelaniwoods** [comment]
@LoganDSPrice that's my mistake! 😞  I made a typo in the description. The first flag should be `--rc=<path to fd>`. There was a different error that I fixed so you can pull to get those changes and try again.

**Comment by LoganDSPrice** [comment]
I was able to generate and run the server. Got "Yay! You're on rails!"
LGTM :shipit:

**Pull request merged**